docstore/driver/codec.go

```diff
@@ -704,11 +724,11 @@ func fieldByIndexCreate(v reflect.Value, index []int) (reflect.Value, bool) {
 }
 
 func decodingError(v reflect.Value, d Decoder) error {
```

Comment:
This is unrelated to this PR but helps debugging significantly, otherwise all the stack traces for errors point here.